packages/appstore/src/components/wallet/wallet.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
let's move all this folder to this directory since this is a wrapper for all wallet components
`packages/appstore/src/containers`